cdap-ui/app/features/tracker/controllers/tags-ctrl.js

```diff
@@ -0,0 +1,94 @@
+/*
+ * Copyright © 2016 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+class TrackerTagsController{
+  constructor($state, $scope, myTrackerApi, $uibModal) {
+    this.$state = $state;
+    this.$scope = $scope;
+    this.myTrackerApi = myTrackerApi;
+    this.$uibModal = $uibModal;
+
+    this.tagList = {
+      preferred: 0,
+      preferredTags: [
+        {
+          'key': 'tag1',
+          'total_entity': 4
+        },
+        {
+          'key': 'tag2',
+          'total_entity': 0
+        },
+        {
+          'key': 'tag3',
+          'total_entity': 39
+        },
+        {
+          'key': 'tag4',
+          'total_entity': 17
+        },
+        {
+          'key': 'tag5',
+          'total_entity': 0
+        }
+      ],
+      user: 0,
+      userTags: [
+        {
+          'key': 'tag1',
```

Comment:
This should be either `name` or navigation to search should use `key`. Right now we get an empty double quotes in the search page in place of name.

<img width="1280" alt="screen shot 2016-07-08 at 8 52 04 am" src="https://cloud.githubusercontent.com/assets/1452845/16693258/45605fee-44e9-11e6-8c90-38f8503afe83.png">